tasks/task04/task04.py

```diff
@@ -0,0 +1,30 @@
+tree = {
+   "node1": {
+       "node11": {
+           "node111": [1, 2, 3],
+           "node112": [4, 5]
+       },
+       "node12": [6],
+       "node13": [13]
+   },
+   "node2": [7, 8, 9]
+}
+
+def collect_leaves(tree):
+  result = []
+  if isinstance(tree, list):
+    for x in tree:
+      result.append(x)
+  if isinstance(tree, dict):
+    for k, v in tree.items():
```

Comment:
if you don't use the key in your code, you can use `tree.values()` and write `for value in tree.values()`

Reply:
thanks